wp-content/themes/ipbs-largo/less/mixins.less

```diff
@@ -1,3 +1,18 @@
+.widgettitle-mixin() {
```

Comment:
Can we add a `text-align: center;` to this mixin?

Reply:
Sounds like a good idea.

Reply:
added in https://github.com/INN/umbrella-ipbs/pull/72/commits/093a46fb4bed32d8448b6c268a2a474ceb93226c